configure.ac

```diff
@@ -487,34 +487,35 @@ AS_IF([test x$have_gui = xyes],
 [
   # settings for Qt
   LIBS="$LIBS $QT_LIBS"
-  PKG_FLAGS="$PKG_FLAGS $QT_CFLAGS"
```

Comment:
Why did you remove this line?

This causes the error mentioned here: https://github.com/SoundScapeRenderer/ssr/pull/58#issuecomment-272497507